### DISCUSSION TITLE
Authentication failed

### DISCUSSION THREAD

**Discussion by arzuunrk** [community/community#58410] [New to GitHub]
Authentication failed. Some common reasons include:

- You are not logged in to your account: see GitHub Desktop > Preferences.
- You may need to log out and log back in to refresh your token.
- You do not have permission to access this repository.
- The repository is archived on GitHub. Check the repository settings to confirm you are still permitted to push commits.
- If you use SSH authentication, check that your key is added to the ssh-agent and associated with your account.
- If you use SSH authentication, ensure the host key verification passes for your repository hosting service.
- If you used username / password authentication, you might need to use a Personal Access Token instead of your account password. Check the documentation of your repository hosting service--

 I have been encountering the "Authentication failed" error repeatedly, even after going through all the troubleshooting steps recommended by GitHub. I have carefully checked the following:

I am logged in to my GitHub account and my credentials are correct.
I have logged out and logged back in to refresh my token.
I have the necessary permissions to access the repository.
The repository is not archived on GitHub, and I should still be able to push commits.
I am using SSH authentication, and my key is added to the ssh-agent and associated with my account.
Host key verification is passing for the repository hosting service.
I have even tried using a Personal Access Token instead of my account password for authentication.
Despite all these efforts, I continue to receive the "Authentication failed" error. This issue is causing a significant hindrance to my workflow, and I would greatly appreciate any assistance in resolving it.

Thank you in advance for your help.

**Comment by queenofcorgis** [comment]
Hi there 👋🏼 

Are you still experiencing this?

**Reply by kwangshi** [reply]
I have the same problem. I just could not find solution.

**Comment by Frid0l1n** [comment]
Try to generate a ssh-key, on linux and windows you could try ssh-keygen and then open the path where the public ssh key is generated open the file and add it to your ssh-key, it should work. You should use the ssh.pub key

**Comment by vishalkrishna8** [comment]
Hey @arzuunrk , here are some steps to troubleshoot the issue:
1.Try a Personal Access Token (PAT) instead of a password.
2.Check your repository permissions.
3.Verify network and firewall settings.
Let me know if this was helpful.
